### PR TITLE
use more self-explanatory variable names

### PR DESCRIPTION
**Pull request by k-nut**
Maybe you have a good reason for your very short variable names but I think that opting for longer ones makes it much more easily understandable. 

I'm pretty sure that most of them are called what they represent now but it would probably be best for you to check again 🔎 

I opted to keep `r` instead of `radius` as that is quite a common abbreviation.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,27 +23,26 @@ const coords = (r, b) => ({
...
-const score = (m, n, r) => Math.abs(findA(m, n, r)*n/r-(5/7))
+const score = (m, n, r) => Math.abs(calculateSeatDistance(m, n, r)*n/r-(5/7))
+
+const calculateNumberOfRings = (seatCount, r) => {
+	let n = Math.floor(Math.log(seatCount)/Math.log(2)) || 1
+	let distance = score(seatCount, n, r)
 
-const findN = (m, r) => {
-	let n = Math.floor(Math.log(m)/Math.log(2)) || 1
-	let a = findA(m, n, r)
```
Comment (k-nut) [inline index.js:38]: `a` was not used at all

File: index.js [hunk 2]
```diff
@@ -63,48 +62,43 @@ const nextRing = (rings, ringProgress) => {
...
 			point = coords(r, j*a)
-			point.r = 0.4*a0
+			point.r = 0.4*seatDistance
 			ring.push(point)
 		}
 		points.push(ring)
 	}
 
 	// fill seats
-	let initial = true
```
Comment (k-nut) [inline index.js:107]: `initial` also was not used

File: index.js (hunk 1 above)
Comment (derhuerst) [reply index.js:38]: A linter might be useful to catch these.

**Comment by juliuste** [comment]
Thanks for the PR! Quick explanation on how the variables got their crazy names: Before I started coding this module, I tried to figure stuff out on my whiteboard. That's why I originally used mathematical variable names, also including indices (like `a0`), and keeping those names at first made translating the algorithm into actual code much easier. (I wasn't quite sure on how to implement it, that's also the reason why there were unused variables like `initial`.)
That being said, now that the script is working, I totally agree that it's much better to use meaningful names, so thanks for the help (:

**Pull request merged**